apps/central-scan/backend/src/fujitsu_scanner.test.ts

```diff
@@ -93,14 +93,14 @@ test('fujitsu scanner returns ballot audit id on scans when imprinting', async (
   await expect(sheets.scanSheet()).resolves.toBeUndefined();
 });
 
-test('fujitsu scanner can scan with letter size', () => {
+test('fujitsu scanner can scans with expected params on letter size election', () => {
```

Comment:
Picking between "can scan" and "scans".
```suggestion
test('fujitsu scanner scans with expected params on letter-size election', () => {
```